sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -145,9 +145,9 @@ The following section provides several code snippets illustrating common pattern
 
 These code samples show common champion scenario operations with the Azure Event Grid client library.
 
-* Publish Custom Events to a topic: [cs1_publish_custom_events_to_a_topic.py][python-eg-sample-customevent]
-* Publish Custom events to a domain topic: [cs2_publish_custom_events_to_a_domain_topic.py][python-eg-sample-customevent-to-domain]
-* Publish a Cloud Event: [cs6_publish_events_using_cloud_events_1.0_schema.py][python-eg-sample-send-cloudevent]
+* Publish Custom Events to a topic: [sample_publish_custom_events_to_a_topic.py][python-eg-sample-customevent]
+* Publish Custom events to a domain topic: [sample_publish_custom_events_to_a_domain_topic.py][python-eg-sample-customevent-to-domain]
```

Comment:
I think these two may have been renamed/changed/removed since EG no longer has a CustomEvent class?
```suggestion
* Publish EventGrid Events to a topic: [sample_publish_eg_events_to_a_topic.py][python-eg-sample-egevent]
* Publish EventGrid Events to a domain: [sample_publish_eg_events_to_a_domain.py][python-eg-sample-egevent-to-domain]
```